### ISSUE TITLE
Doc inconsistency and fatal error

### ISSUE THREAD

**Issue by alex-moreno** [FloeDesignTechnologies/phpcs-security-audit#15]
I get this:

```Fatal error: Uncaught Error: Call to undefined method Security_Sniffs_Utils::drupal_parse_info_format() in /Users/alejandromoreno/projects/[...]/vendor/pheromone/phpcs-security-audit/Security/Sniffs/Drupal7/AdvisoriesContribSniff.php:34```

but apart of that, the  --standard=example_drupal7_ruleset.xml does not work, it points to use Security instead:

```./vendor/bin/phpcs --standard=Security --extensions=php,module,inc,install,test,profile,theme,css,info,txt,md docroot/sites/all/modules/custom/```

Thanks in advance

**Comment by alex-moreno** [comment]
```
 ./vendor/bin/phpcs  --version
PHP_CodeSniffer version 2.9.1 (stable) by Squiz (http://www.squiz.net)```

**Comment by alex-moreno** [comment]
ok, got this fixed. symlink does not seem to work well, a temporary solution was to force moving the folder:

```
sudo mv vendor/pheromone/phpcs-security-audit/Security/ vendor/squizlabs/php_codesniffer/CodeSniffer/Standards/Security
```

instead of the ln -s which, again, does not seem to work.

**Comment by alex-moreno** [comment]
the doc is consistent btw.

**Comment by alex-moreno** [comment]
related to this issue: https://github.com/FloeDesignTechnologies/phpcs-security-audit/issues/2

**Comment by jmarcil** [comment]
I've fixed in 1d25d9bb862c6103f34c18f76542249e6bd17a69 the dependency that might mess up things if installed from Packagist directly: `"squizlabs/php_codesniffer": "~1.5.1 || >=2.3.3 <3.0"`
